index.js

```diff
@@ -172,6 +172,7 @@ const replyWithDetailsForRoundNumber = async (res, client, roundNumber) => {
 
   json(res, {
     roundId: round.id.toString(),
+    maxTasksPerNode: round.max_tasks_per_node,
```

Comment:
would it be more useful to return `delayBetweenRetrievals` (in ms) here? Since we know the round length, we can let the client be simpler and calculate this here

Reply:
It was my initial thought, too. But: while `delayBetweenRetrievals` is simpler in the short-term, it's something we will need to rework in the longer term as we progress towards decentralised protocol as described in our design docs.

Also spark-api knows as little about the round length as spark-checker does. 

In the future, I'd like spark-checkers to query on-chain state to determine when a new round starts and how many epochs/blocks remain in the current round. That will produce a much more accurate value than spark-api can ever provide.

I am imagining that in each iteration, spark-checker can:
1. Find out what's the current round index and when the next round is should start (see [`currentRoundEndBlockNumber`](https://github.com/Meridian-IE/impact-evaluator/blob/156ad2506b99db9e951f4444384e1a94958ea1cd/src/ImpactEvaluator.sol#L9))
2. Determine how many tasks we have completed this round.
3. If too many - then wait for the next round before doing any more work
4. If less than maxTasksPerNode, then do another retrieval
5. Decide how long to wait before the next iteration depending on how much time is remaining in the current round and how many more tasks we can run until we reach the max